client/blocks/image-editor/index.jsx

```diff
@@ -21,7 +21,7 @@ import ImageEditorCanvas from './image-editor-canvas';
 import ImageEditorToolbar from './image-editor-toolbar';
 import ImageEditorButtons from './image-editor-buttons';
 import MediaUtils from 'lib/media/utils';
-import closeOnEsc from 'lib/mixins/close-on-esc';
+import closeOnEscHoc from 'lib/mixins/close-on-esc/hoc';
```

Comment:
I worry that abbreviating `hoc` is very unclear for new developers. I'd rather we at the very least named this `closeOnEsc`. Ideally the mixin would be dropped altogether and a higher-order component is the default export of `lib/close-on-esc`.